fjord/feedback/tests/test_api.py

```diff
@@ -926,10 +924,12 @@ def data_generator():
         # created
         for i in range(throttle_trigger):
             # django-ratelimit fails the throttling if we hit the url
-            # a fajillion times in rapid succession. For now, we add
-            # a sleep which means this test takes 5 seconds now.
+            # a fajillion times in rapid succession. For now, we add a
+            # sleep which makes this test run a little longer than it
+            # probably needs to.
+            #
             # FIXME: Look into this more for a better solution.
-            time.sleep(0.05)
+            time.sleep(0.01)
```

Comment:
0.01 works locally. If I remove this altogether, it fails. I didn't want to take the time to look into it. If it fails on travis, I'll increase this again.